webapp/channels/src/components/pdf_preview.tsx

```diff
@@ -130,16 +149,16 @@ export default class PDFPreview extends React.PureComponent {
             return;
         }
 
-        const page = await this.loadPage(this.state.pdf, pageIndex);
+        const page = await this.loadPage(this.state.pdf!, pageIndex);
         const context = canvas.getContext('2d');
         const viewport = page.getViewport({scale: this.props.scale});
         canvas.height = viewport.height;
         canvas.width = viewport.width;
 
         const renderContext = {
-            canvasContext: context,
+            canvasContext: context as object,
             viewport,
-        };
+        } as RenderParameters;
```

Comment:
Why do we need 2 types assertions here?

Reply:
Initially i did not add this ! after seeing the ci fail i added the type assertion!
https://github.com/mattermost/mattermost/actions/runs/6691861304/job/18184963043#step:4:13
![image](https://github.com/mattermost/mattermost/assets/93595710/ddfdaaa8-1d9b-472d-b427-ab691c77700d)

Reply:
>https://github.com/mattermost/mattermost/actions/runs/6691861304/job/18184963043#step:4:13 ![image](https://user-images.githubusercontent.com/93595710/279264300-ddfdaaa8-1d9b-472d-b427-ab691c77700d.png)

 Initially i did not add this ! after seeing the ci fail i added the type assertion!